src/Language/PureScript/TypeChecker/Kinds.hs

```diff
@@ -588,10 +588,8 @@ type DataDeclarationArgs =
   )
 
 type DataDeclarationResult =
-  ( [(DataConstructorDeclaration, SourceType)]
-  -- ^ The infered type signatures of data constructors
-  , SourceType
-  -- ^ The inferred kind of the declaration
+  ( [(DataConstructorDeclaration, SourceType)] -- The infered type signatures of data constructors
```

Comment:
I'd prefer like this like it was before, so that the lines aren't too long.
```suggestion
  ( [(DataConstructorDeclaration, SourceType)]
    -- The infered type signatures of data constructors
```